pkg/antctl/raw/helper.go

```diff
@@ -146,3 +151,26 @@ func CreateControllerClientCfg(k8sClientset kubernetes.Interface, antreaClientse
 	cfg.Host = fmt.Sprintf("https://%s", net.JoinHostPort(nodeIP, fmt.Sprint(controllerInfo.APIPort)))
 	return cfg, nil
 }
+
+func AddSchemeToClientCfg(kubeconfig *rest.Config) (client.Client, error) {
+	var err error
+	scheme := k8sruntime.NewScheme()
```

Comment:
@tnqn editing on other comments, but for this one, I know the meaning, but `init()` cannot throw out an error, at first we wanted to throw out error on this, so do we think it is not that important to throw out an error?

Reply:
Initializing a schema is a static routine which has no external dependency, it either throws error because of a programming error which should be identified during development or never throw error. In many code it's wrapped with `utilruntime.Must()` to ensure it never fails.
https://github.com/antrea-io/antrea/blob/1d27432bfdbc5a25a790b85fd562050505d68bb1/pkg/client/clientset/versioned/fake/register.go#L65